.github/workflows/coverage.yml

```diff
@@ -0,0 +1,46 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Huawei Inc.
+#
+name: Generate Coverage Badge
```

Comment:
pls change the name to - `xpanse-test-coverage`

Reply:
ok